Makefile

```diff
@@ -31,7 +33,14 @@ validate-with-json: $(all-validations-with-json)
 .PHONY: %.json.test
 %.json.test:
 	mkdir -p logs
-	$(TEST) $(subst _json/,.wdl, $(dir $(basename $@))) -i $(basename $@) -o test_options.json 2>&1 | tee logs/$(notdir $(subst _json/,.wdl, $(dir $(basename $@))))_with_$(notdir $(basename $@)).log 
+	$(TEST) $(subst _json/,.wdl, $(dir $(basename $@))) -i $(basename $@) -o test_options.json 2>&1 | tee logs/$(notdir $(subst _json/,.wdl, $(dir $(basename $@))))_with_$(notdir $(basename $@)).log ; \
+	test_result=$${PIPESTATUS[0]} ; \
```

Comment:
And this is what made me stumble upon this pr again.  Slightly different (simpler?) solution also in https://github.com/broadinstitute/palantir-workflows/pull/116.  Any thoughts on a preference between them?

Reply:
I rebased it to your solution